### DISCUSSION TITLE
Elementor Pro conflict with BuddyPress plugin

### DISCUSSION THREAD

**Discussion by MissJimJam** [elementor/elementor#17757] [Developers Community]
I am a total novice with zero experience so am probably posing this question in the wrong place, but in case someone can help...

When I try to edit a page with Elementor and I have the BuddyPress plugin active, I get the following error "The Preview Could Not Be Loaded" with the below debug info.... not sure if this means anything to anyone, but I need both Elementor and BuddyPress. And no idea how I can resolve this. Any help appreciated.

Fatal error: Uncaught TypeError: Argument 1 passed to Elementor\Controls_Stack::sanitize_settings() must be of the type array, null given, called in /home/customer/www/sheisaction.com/public_html/wp-content/plugins/elementor/includes/base/controls-stack.php on line 1099 and defined in /home/customer/www/sheisaction.com/public_html/wp-content/plugins/elementor/includes/base/controls-stack.php:2338 Stack trace: #0 /home/customer/www/sheisaction.com/public_html/wp-content/plugins/elementor/includes/base/controls-stack.php(1099): Elementor\Controls_Stack->sanitize_settings(NULL) #1 /home/customer/www/sheisaction.com/public_html/wp-content/plugins/elementor/includes/base/controls-stack.php(2087): Elementor\Controls_Stack->get_data('settings') #2 /home/customer/www/sheisaction.com/public_html/wp-content/plugins/elementor/core/base/base-object.php(142): Elementor\Controls_Stack->get_init_settings() #3 /home/customer/www/sheisaction.com/public_html/wp-content/plugins/elementor/core/base/base-object.php(40): Elementor\Core\Base\Ba in /home/customer/www/sheisaction.com/public_html/wp-content/plugins/elementor/includes/base/controls-stack.php on line 2338
Uncaught TypeError: Argument 1 passed to Elementor\Controls_Stack::sanitize_settings() must be of the type array, null given, called in /home/customer/www/sheisaction.com/public_html/wp-content/plugins/elementor/includes/base/controls-stack.php on line 1099 and defined in /home/customer/www/sheisaction.com/public_html/wp-content/plugins/elementor/includes/base/controls-stack.php:2338
Stack trace:
#0 /home/customer/www/sheisaction.com/public_html/wp-content/plugins/elementor/includes/base/controls-stack.php(1099): Elementor\Controls_Stack->sanitize_settings(NULL)
#1 /home/customer/www/sheisaction.com/public_html/wp-content/plugins/elementor/includes/base/controls-stack.php(2087): Elementor\Controls_Stack->get_data('settings')
#2 /home/customer/www/sheisaction.com/public_html/wp-content/plugins/elementor/core/base/base-object.php(142): Elementor\Controls_Stack->get_init_settings()
#3 /home/customer/www/sheisaction.com/public_html/wp-content/plugins/elementor/core/base/base-object.php(40): Elementor\Core\Base\Ba

**Comment by G3zeck** [comment]
hello, can you resolve it?

**Reply by G3zeck** [reply]
I have the same problem

**Comment by toineenzo** [comment]
Please share system info
